recipes/wradlib/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.7.0" %}
+
+package:
+    name: wradlib
+    version: {{ version }}
+
+source:
+    fn: wradlib-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/w/wradlib/wradlib-{{ version }}.tar.gz
+    md5: cb3b3f07b4cd4e31f7bd4018013e3f29
+
+build:
+    number: 0
+    # wradlib 0.7.0 does not support Python 3
+    skip: True # [py3k]
+
+requirements:
+    build:
+        - python
+        - numpy >=1.7.0
+        - scipy >=0.9
+        - matplotlib >=1.1.0
+        - h5py >=2.0.1
+        - netcdf4 >=1.0
+        - gdal
+        # The gdal package on OS X is missing a dependency on krb5
+        - krb5  # [osx]
```

Comment:
More precisely, is it something like libgomp (from the top of my head)? That is certainly one I've seen on the ```netcdf-fortran``` package built by IOOS.